### PR TITLE
Add Flixbus station for croatia

### PR DESCRIPTION
**Pull request by noplay**

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -27412,3 +27412,93 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 29559;Vatra Dornei;vatra-dornei;;;47.3447576;25.3521039;;RO;Europe/Bucarest;t;f;f;t;f;f;;;f;;f;;f;;f;;f;15778;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29560;Zalău;zalau;;;47.1854562;23.0573324;;RO;Europe/Bucarest;t;f;f;t;f;f;;;f;;f;;f;;f;;f;15788;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29561;Țăndărei;tandarei;;;44.6488718;27.6438672;;RO;Europe/Bucarest;t;f;f;t;f;f;;;f;;f;;f;;f;;f;15698;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+29562;Bale;bale;;;45.040703;13.783752;;HR;Europe/Zagreb;t;f;f;t;f;f;;;f;;f;;f;;f;;f;9618;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```
Comment (avandecreme) [inline stations.csv]: Will it be clear that it it not Basel, Switzerland in French?

Comment (noplay) [reply stations.csv]: Updated to add the country_hint

File: stations.csv [hunk 2]
```diff
@@ -26092,8 +26092,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
 28227;Šibenik;sibenik;7876760;;43.730039;15.898106;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800017;f;;f;;f;6498;t;;f;;f;;f;;;;f;;f;;f;;f;HRADN;f;f;;;;;;Sebenico;;;;;;シベニク;시베니크;Szybenik;;Шибеник;;;希贝尼克
 28228;Kotoriba;kotoriba;7874808;;46.353613;16.808461;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800057;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;HRADX;f;f;;;;;;;;;;;;;;;;;;;
-28229;Opatija-Matulji;opatija-matulji;7875611;;45.364809;14.321775;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800002;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;HROPA;f;f;;;;;;;;;;;;;;;;;;;
-28230;Pula;pula;7877417;;44.879706;13.847199;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800003;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;HRPUY;f;f;;;;;;Pola;;;;;;プーラ;풀라;;;Пула;;;普拉
+28229;Opatija-Matulji;opatija-matulji;7875611;;45.364809;14.321775;29610;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800002;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;HROPA;f;f;;;;;;;;;;;;;;;;;;;
+28230;Pula;pula;7877417;;44.879706;13.847199;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800003;f;;f;;f;8988;t;;f;;f;;f;;;;f;;f;;f;;f;HRPUY;f;f;;;;;;Pola;;;;;;プーラ;풀라;;;Пула;;;普拉
```
Comment (avandecreme) [inline stations.csv]: What is that added id? Why not in the same place as the others?

Comment (noplay) [reply stations.csv]: Their is a new station: Opatija and I have updated Opatija-Matulji to set it as a child.

**Comment by immae** [comment]
Sorry, you’ll need to rebase because of https://github.com/trainline-eu/stations/pull/496 (new columns added for westbahn)

**Comment by noplay** [comment]
It's rebase

**Pull request merged**